src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMemberTranslatorPlugin.cs

```diff
@@ -1,126 +1,125 @@
+using System;
 using System.Collections.Generic;
-using System.Linq.Expressions;
-using GeoAPI.Geometries;
-using Microsoft.EntityFrameworkCore.Query.Expressions;
-using Microsoft.EntityFrameworkCore.Query.ExpressionTranslators;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.EntityFrameworkCore.Query;
+using Microsoft.EntityFrameworkCore.Query.SqlExpressions;
+using Microsoft.EntityFrameworkCore.Storage;
 using NetTopologySuite.Geometries;
 
 // ReSharper disable once CheckNamespace
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators.Internal
 {
     public class NpgsqlNetTopologySuiteMemberTranslatorPlugin : IMemberTranslatorPlugin
     {
-        public virtual IEnumerable<IMemberTranslator> Translators { get; } = new IMemberTranslator[]
+        public NpgsqlNetTopologySuiteMemberTranslatorPlugin(
+            IRelationalTypeMappingSource typeMappingSource, ISqlExpressionFactory sqlExpressionFactory)
```

Comment:
Let's put each parameter on a new line or keep them on the same line as the method name.